### DISCUSSION TITLE
File and Folder addition/editing through Online Editor

### DISCUSSION THREAD

**Discussion by mertkabadayii** [ls1intum/Artemis#7589] [Q&A]
Hello, Files and Folders, that are editted and added from the Online Editor, are not updated on the projekt repository folder. How can students add/edit Files from Online Editor and make a change in the repository folder?

photo 1 - Online Editor: (newfile and addedfile.cpp)
![online editor](https://github.com/ls1intum/Artemis/assets/78808727/d2fc844c-16b4-41ab-840f-9cf546eb6429)

photo 2 - Gitlab Repository:
![repository](https://github.com/ls1intum/Artemis/assets/78808727/098d5946-17af-46af-a6ee-d824eb3d3465)

**Comment by MaximilianAnzinger** [comment]
Did you submit the changes? (Green button on the right of the editor)
